src/plugins/graph/imports/components/axis/components/axis-or-legend-attribute-menu.tsx

```diff
@@ -42,10 +42,14 @@ const _AxisOrLegendAttributeMenu = ({ place, target, portal,
   const [labelText, setLabelText] = useState(attribute?.name);
   const removeAttrItemLabel = t(removeAttrItemLabelKeys[role], {vars: [attribute?.name]});
   const treatAs = dataConfig?.attributeType(role) === "numeric" ? "categorical" : "numeric";
-  const overlayStyle: CSSProperties = { position: "absolute", ...useOverlayBounds({target, portal}) };
-  const buttonStyle: CSSProperties = { position: "absolute", width: "100%", height: "100%", color: "transparent" };
   const menuRef = useRef<HTMLDivElement>(null);
   const onCloseRef = useRef<() => void>();
+  const overlayStyle: CSSProperties = {
+    position: "absolute", ...useOverlayBounds({target, portal})
+  };
+  const buttonStyle: CSSProperties = {
+    position: "absolute", width: "100%", height: "100%", color: "transparent", left: 0, top: 0
+  };
```

Comment:
Since you are still making changes, here is a version of this which is a little better:
```suggestion
  const buttonStyle: CSSProperties = {
    position: "absolute", inset: 0, padding: 0, color: "transparent"
  };
```
`inset` is short for `top`, `right`, `bottom`, `left` like `margin` is short for `margin-top`...
Setting right and bottom to 0 is the same as setting the width and height to 100%
The button element was inheriting the padding from from the browser's default button which looked weird, setting the padding to 0 makes the button really fill the whole area of the overlay.

Reply:
Oh that's much nicer - thanks - I put in the change.